cmake/thirdparty/FindMFEM.cmake

```diff
@@ -27,8 +27,8 @@ if(EXISTS ${_mfem_cmake_config})
     include(${_mfem_cmake_config})
 
     # TODO: This needs to be verified
```

Comment:
If you verified this works could you remove my long ignored TODO =)

Reply:
This was an oversight. I remove it.